styles.css

```diff
@@ -527,3 +527,102 @@ body {
   bottom: 74.49%;
   opacity: 0.5;
 }
+
+.contact-section {
+  position: absolute;
+  width: 100%;
```

Comment:
- [ ] Please make sure you layout your design using `flexbox` as stated in the project requirement 🙏

Reply:
Fixed